test/org/sagebionetworks/bridge/play/controllers/ScheduledActivityControllerTest.java

```diff
@@ -186,4 +229,24 @@ public void mustBeAuthenticated() throws Exception {
         controller.getScheduledActivities(DateTime.now().toString(), null, null);
     }
     
+    @Test
+    public void oldSessionsWithIdAndNoAccountCreatedOn() throws Exception {
+        session.getUser().setAccountCreatedOn(null); // this is not currently in the session
+        
+        controller.getScheduledActivities(null, "-07:00", "3");
+        verify(scheduledActivityService).getScheduledActivities(any(), contextCaptor.capture());
+        ScheduleContext context = contextCaptor.getValue();
+        assertEquals(ACCOUNT_CREATED_ON, context.getAccountCreatedOn());
+    }
+    
+    @Test
+    public void oldSessionsWithNoIdAndNoAccountCreatedOn() throws Exception {
+        session.getUser().setAccountCreatedOn(null); // these are not currently in the session
+        session.getUser().setId(null);
+        
+        controller.getScheduledActivities(null, "-07:00", "3");
+        verify(scheduledActivityService).getScheduledActivities(any(), contextCaptor.capture());
+        ScheduleContext context = contextCaptor.getValue();
+        assertNotNull(context.getAccountCreatedOn()); // this is a timestamp, so
+    }
```

Comment:
Should we also have a test case where the accountCreatedOn is in the session, and we verify it's being passed correctly to the scheduledActivityService?